app/controllers/newflow/student_signup_controller.rb

```diff
@@ -16,9 +16,11 @@ def student_signup
         contracts_required: !contracts_not_required,
         client_app: get_client_app,
         user_from_signed_params: session[:user_from_signed_params],
+        is_BRI_book: is_BRI_book_adopter?,
         success: lambda {
           save_unverified_user(@handler_result.outputs.user.id)
-          security_log :student_signed_up, { user: @handler_result.outputs.user }
+          security_log(:student_signed_up, { user: @handler_result.outputs.user })
+          clear_cache_BRI_marketing
```

Comment:
On student and educator flow, clear_cache_BRI_marketing is called on success. On social_outh flow, it is called on success and failure. Is this intentional?

Reply:
I don't believe it is called on failure.

Reply:
You're right. I didn't pay attention to the line numbers.